config/locales/ar-LB.yml

```diff
@@ -904,7 +948,6 @@ ar-LB:
       dash_cases_by_workflow: "عرض الحالات بحسب سير العمل"
       dash_cases_by_task_overdue: "عرض الحالات بحسب المهمّات المتأخّرة"
       dash_cases_to_assign: "عرض الحالات التي تستلزم التكليف"
-      dash_show_none_values: "عرض من دون القيم"
```

Comment:
How did this get removed? @eyemails , can you make sure this translation gets added back in Transifex, then re-download this file from Transifex and update the PR?